css/projects.css

```diff
@@ -545,3 +545,42 @@ footer {
     color: #ADADAD;
   }
 }
+
+/* project card */
+
+#fav-count{
+  margin-bottom: 10px;
+}
+
+#fav-count-text{
+  color:black;
+  font-size: 18px;
+}
+
+.project-list {
+  position: relative;
+  box-sizing: border-box;
+  width:100%;
+  overflow:hidden;
+  padding-bottom: calc( 15px + 1rem*1.3 );
+  margin-left: 10px;
+  margin-right: 10px;
+  transition:left .25s, height .25s, padding .25s, margin .25s;
+  /* optimize swipe with class and transform. */
+}
```

Comment:
this can be simplifed 😉 you'll probably only need to keep `width`, `margin-left`, and  `margin-right`.